corehq/apps/data_cleaning/views/tables.py

```diff
@@ -127,8 +132,18 @@ def select_all(self, request, *args, **kwargs):
 
     @hq_hx_action("post")
     def apply_all_changes(self, request, *args, **kwargs):
-        # todo: this is a placeholder
-        return self.get(request, *args, **kwargs)
+        self.session.prepare_session_for_commit()
+        commit_data_cleaning.apply_async(
+            args=(self.session.session_id,),
+            task_id=self.session.task_id
```

Comment:
What's the importance of being able to pass in the `task_id` here? Or this just ordered this way as opposed to starting the task here and having to retrieve and set `self.session.task_id` afterward?

Reply:
so that you can poll the status. it's a celery task id